charts/ldap/README.md

```diff
@@ -1,6 +1,6 @@
 # README
 
-This helm chart installs the jenkins infrastructure ldap service (ldap.jenkins.io) and is designed to be working with the [jenkinsciinfra/openldap](https://github.com/jenkinsciinfra/openldap) docker image.
+This helm chart installs the jenkins infrastructure ldap service (ldap.jenkins.io) and is designed to be working with the [jenkins-infra/ldap](https://github.com/jenkins-infra/ldap) docker image.
```

Comment:
the link seems to have been intended to be https://hub.docker.com/r/jenkinsciinfra/ldap not that there's any info there?

Reply:
oh maybe yea, i'll leave it over night and see if anyone else knows what it should be, right now its not working.

Reply:
this is probably better anyway

Reply:
I expect the link to DockerHub to be part of the image's readme, so this change here is good ✅